installer/test/installer_test.exs

```diff
@@ -0,0 +1,8 @@
+defmodule InstallerTest do
```

Comment:
no need for this test

Reply:
I will remove this was autogenerated by mix new script